src/test/java/com/gooddata/warehouse/WarehouseServiceIT.java

```diff
@@ -30,11 +31,13 @@
 public class WarehouseServiceIT extends AbstractGoodDataIT {
 
     private static final String TITLE = "Test";
+    private static final String SCHEMA_NAME = "default";
```

Comment:
yes, default schema always exists in warehouse.  Added method `getDefaultWarehouseSchema` to the service.